### PR TITLE
Markdown to RST docs configuration

### PR DESCRIPTION
**Pull request by LiraLemur**
### Description of the Change

We are fixing the docs so .md's from the repo could be integrated into ReadTheDocs. 
Current PR is making it work and also adds one of such files into Contributing section. 

### Benefits

1. Less outdated docs - as soon as .md's change docs will change accordingly. 
2. Easy to put existing md docs into ReadTheDocs

### Possible Drawbacks 

Not everything transfers decently from md to rst. Though, when such files are used in docs, one can easily check the way it looks. 
In short, 100% pretty docs VS. Updated docs

### Usage Examples or Tests 

Docs can be included directly into the documentation files like this: 
`.. mdinclude:: ../../../CONTRIBUTING.md`

### Alternate Designs 

Transfering such files manually from .md to .rst: easy to miss something and very time-consuming

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -30,7 +31,7 @@
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ['.rst', '.md']
```
Comment (igor-egorov) [inline docs/source/conf.py]: The whole comment above can be removed.

Comment (LiraLemur) [reply docs/source/conf.py]: It was there originally. Maybe let's leave it there for convenience in the future?..

**Comment by LiraLemur** [comment]
> Thanks!
> 
> Internal links on the Contributing page are broken :(
> image

@igor-egorov that's odd - "Contact developers" worked, that's what I've tried. Maybe you could check for the change of the pattern there?

File: docs/source/conf.py (hunk 1 above)
Comment (neewy) [reply docs/source/conf.py]: We can remove it :)

Comment (igor-egorov) [reply docs/source/conf.py]: woohooa)
